scripts/jenkins-build.bat

```diff
@@ -132,3 +132,48 @@ for %%T in (bbbc mitocheck tubhiswt) do (
         install\bin\pixels-performance 1 !input! %outdir%\!test!-cpp.ome.tiff %resultsdir%\!test!-pixeldata-win-cpp-%%I.tsv
```

Comment:
As mentioned on the [trello card](https://trello.com/c/v0jZwWJv/13-scripts-and-naming) I think here it would be useful to put the new logic into a separate batch file (e.g. `tiling.bat`), move the other logic into `metadata.bat` and `pixeldata.bat`, and then have `jenkins-build.bat` call `tiling.bat`.  The common logic at the top of `jenkins-build.bat` can stay as it is--this is just splitting out the test invocation so we can select the tests we want to run for each benchmarking run; for tiling we don't need to run the other tests.